# Message to specify the new goal position
geometry_msgs/Pose goalPose
---